core/pfe-sass/maps/_typography_deprecated.scss

```diff
@@ -0,0 +1,29 @@
+@use "@patternfly/patternfly/sass-utilities/all" as *;
+@use "../variables/4-sizes" as sizes;
+
+$pfe-typography-base-deprecated: (
```

Comment:
What are your thoughts on removing this?  It is marked for deprecation but I worry about debugging instances where we were relying on this output.  For whatever reason that might be.